### DISCUSSION TITLE
maximum dt for wall turbulence

### DISCUSSION THREAD

**Discussion by yumilili** [xcompact3d/Incompact3d#268] [Q&A] [answered]
For DNS of TBL, what is the possible maximum dt that I can take? Could I choose dt that gives CFL_y = 1 (CFL_x and CFL_z are much smaller than CFL_y)? Or should I choose dt based on the viscous time scale, i.e., dt+ = 0.1?

**Comment by slaizet** [comment, marked as answer]
Hi,
I would not recommend using CFL conditions to find the optimal time step as the stability of the simulations. I would suggest a test-and-trial error instead. The CFL is here purely indicative.
Best,
Sylvain
